Q/qwtw/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qwtw"
+version = v"2.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ig-or/qwtw.git", "b9451c91004de517e97429cd762caaad206f6014")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd qwtw
+mkdir build
+cd build
+export LD_LIBRARY_PATH=/workspace/srcdir/qwtw/build:$LD_LIBRARY_PATH
+ln -s /lib64/libc.so.6 libc.so
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ../.
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libqwtw", :qwtw),
+    ExecutableProduct("qwproc", :qwproc)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="Libglvnd_jll", uuid="7e76a0d4-f3c7-5321-8279-8d96eeed0f29"))
+    Dependency(PackageSpec(name="boost_jll", uuid="28df3c45-c428-5900-9ff8-a3135698ca75"))
+    Dependency(PackageSpec(name="Qt_jll", uuid="ede63266-ebff-546c-83e0-1c6fb6d0efc8"))
+    Dependency(PackageSpec(name="qwt_jll", uuid="ed0789fa-10db-50b3-94da-03266d70be0f"))
+]
+
+Pkg.add("Qt_jll")
+using Qt_jll
+ENV["QT_PLUGIN_PATH"]=Qt_jll.artifact_dir*"/plugins"
+ENV["LD_LIBRARY_PATH"] = string(Qt_jll.LIBPATH) * ":" * ENV["LD_LIBRARY_PATH"]
```

Comment:
What?  Why this?

Reply:
how 100% sure this is needed right now..  need your advice on this maybe

explanation (sorry for many letters):  previous version of the library used to start QT thread from inside the library, right after the library is loaded (in a separate thread, though); BinaryBuilder is testing the library by loading it in separate julia process (dlopen); test was failing because this julia process returns non zero because my library was failing because it could not find QT plugins. After I added this QT_PLUGIN_PATH, it starts working good.

But, now this is a bit different. QT is starting in a separate process, not from inside the library. And the library (libqwtw.so) does not depend on QT.  But the executable qwproc still needs QT.  So I'm not sure, can we remove those lines?

Reply:
These lines must go for sure
```suggestion
```

Reply:
Would you please comment about if BinaryBuilder is trying to run the executable files/products?  If NO, then just remove those lines.   If yes, then libraries/binaries which depend on QT need QT_PLUGIN_PATH (and platform plugin at least).  Or, (as ugly workaround) maybe it's possible to detect (inside the program) that program "was started by BinaryBuilder" and not start QT in this case.

Reply:
BinaryBuilder _must_ be able to load the library alone in an independent process, without this kind of hints.  These lines are even preventing to the build script to actually start, so please remove them.